src/components/prediction/ModelSelector.js

```diff
@@ -0,0 +1,65 @@
+import React, {Component} from 'react';
+import {Card} from 'react-md/lib/Cards/index';
+import SelectField from 'react-md/lib/SelectFields';
+import PropTypes from 'prop-types';
+import {modelsLabel} from '../../propTypes';
+import {Button, CardText} from 'react-md';
+import {CLASSIFICATION, REGRESSION} from '../../reference';
+
+class ModelSelector extends Component {
```

Comment:
actually this would look better with a radio button to select Classification or Regression and then only one select field.

Reply:
But I think the idea was that the user can select both models.